Dockerfile

```diff
@@ -57,6 +57,11 @@ ENV SHELL /bin/bash
 RUN yes |ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Linuxbrew/install/master/install)"
 RUN brew doctor || true
 
+# Manually install protobuf with --no-binary flag before installing protoc
+RUN brew install python
```

Comment:
Got it. Not opposed to this, just wondering why, especially if `brew install grpc` is going to lead to switching to linuxbrew python being installed anyway.

Do you mind noting the rationale in a comment? I'm hoping we can stop installing `grpc` with linuxbrew relatively soon, and it might be worth revisiting this then. (Most of the languages already no longer need it, since the codegen in those languages is done through the per-language `grpc-tools` packages.)